index.html

```diff
@@ -341,7 +341,12 @@ <h3 class="best">
               ></textarea>
             </label>
             <div class="btn__container">
-              <button type="submit" name="submit" class="btn__submit">
+              <button
+                onclick="submitBtnFunction()"
```

Comment:
- [ ] The submit button is working because it seems is not connected 
![image](https://user-images.githubusercontent.com/62705947/202708352-64245041-c240-46ba-84c3-46fb476e459b.png)